src/app/pages/Gallery/ToTopIcon.tsx

```diff
@@ -0,0 +1,25 @@
+type ToTopIconProps = {
+  className: string;
+  onClick: () => void;
+};
+
+function ToTopIcon({ className, onClick }: ToTopIconProps): JSX.Element {
+  return (
+    <svg
+      className={className}
+      onClick={onClick}
+      width="24"
+      height="30"
+      viewBox="0 0 24 30"
+      fill="none"
```

Comment:
I'm not sure if you really need this line, as you specify fill in the path below.

Reply:
👍